src/test/java/net/sf/jabref/logic/l10n/LocalizationConsistencyTest.java

```diff
@@ -124,86 +122,49 @@ public void findMissingLocalizationKeys() throws IOException {
         List<LocalizationEntry> missingKeys = LocalizationParser.find(LocalizationBundle.LANG).stream().sorted()
                 .distinct().collect(Collectors.toList());
 
-        printInfos(missingKeys);
-
-        String resultString = missingKeys.stream().map(Object::toString).collect(Collectors.joining("\n"));
-        assertEquals(
-                "source code contains language keys for the messages which are not in the corresponding properties file",
-                "", resultString);
+        assertEquals("DETECTED LANGUAGE KEYS WHICH ARE NOT IN THE ENGLISH LANGUAGE FILE\n" +
+                "1. PASTE THESE INTO THE ENGLISH LANGUAGE FILE\n" +
+                "2. EXECUTE: gradlew localizationUpdate\n" +
+                missingKeys.parallelStream()
+                        .map(key -> String.format("%s=%s", key.getKey(), key.getKey()))
+                        .collect(Collectors.toList()),
+                Collections.EMPTY_LIST, missingKeys);
```

Comment:
And it's also immuntable: https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#emptyList-- 
(Unlike this method, the field does not provide type safety.)